pyatlan/model/enums.py

```diff
@@ -1890,6 +1892,8 @@ class AtlanIcon(str, Enum):
     YARN = "PhYarn"
     YIN_YANG = "PhYinYang"
     YOUTUBE_LOGO = "PhYoutubeLogo"
+    CRATEDB = "CrateDb"
+    KX = "Kx"
```

Comment:
Are you sure we need to update this? 🤔 
I think we only need to add new connector entries to the `AtlanConnectorType` enum, which you've already done above^^ 

For example
- https://github.com/atlanhq/atlan-python/pull/502/files
- https://github.com/atlanhq/atlan-frontend/pull/23719/files

Reply:
I've removed this for now since I need to cut the SDK release. If we need this, please create a PR in the future. Thanks!